src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -833,15 +845,18 @@ public static AppType appType( ResultSet rs ) throws SQLException {
             data.put( LAST_INDEX_FQN, lastIndex( rs ) );
         }
 
-
         final Set<PropertyType> allPropertyTypes = authorizedPropertyTypes.values().stream()
                 .flatMap( propertyTypesOfEntitySet -> propertyTypesOfEntitySet.values().stream() )
                 .collect( Collectors.toSet() );
         for ( PropertyType propertyType : allPropertyTypes ) {
             List<?> objects = propertyValue( rs, propertyType );
 
             if ( objects != null ) {
-                data.putAll( propertyType.getType(), objects );
+                if ( propertyType.getDatatype() == EdmPrimitiveTypeKind.Binary ) {
+                    data.putAll( propertyType.getType(), byteBlobDataManager.getObjects( objects ) );
```

Comment:
This should return pre-signed URLs and not the actual objects.

Reply:
Same as above